src/Fixer/Alias/RandomApiMigrationFixer.php

```diff
@@ -38,16 +40,24 @@ public function configure(array $configuration = null)
         static $validFunctions = array('rand', 'srand', 'getrandmax');
```

Comment:
don't change the input structure and keep number of indexes in separated structure